cmake/modules/PytestBenchmark.cmake

```diff
@@ -0,0 +1,21 @@
+include (FindPythonModule)
+
+# Check which Python version ROOT was build against  
+find_program(ROOT_CONFIG_EXECUTABLE root-config)
+exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--python-version" OUTPUT_VARIABLE ROOT_PYTHON_VERSION RETURN_VALUE RETVAR)
+set(ROOT_PYTHON_VERSION ${ROOT_PYTHON_VERSION} CACHE INTERNAL "")
+string(REGEX REPLACE "^([0-9]+)\\.[0-9]+.*" "\\1" PYTHON_VERSION_MAJOR "${ROOT_PYTHON_VERSION}")
+
+# FindPython we used to build ROOT against 
+find_package(Python${PYTHON_VERSION_MAJOR} COMPONENTS Interpreter NumPy)
+set(PYTHON_EXECUTABLE "${Python${PYTHON_VERSION_MAJOR}_EXECUTABLE}" CACHE INTERNAL "" FORCE)
+set(PYTHON_INCLUDE_DIRS "${Python${PYTHON_VERSION_MAJOR}_INCLUDE_DIRS}" CACHE INTERNAL "" FORCE)
+set(PYTHON_LIBRARIES "${Python${PYTHON_VERSION_MAJOR}_LIBRARIES}" CACHE INTERNAL "" FORCE)
+set(PYTHON_VERSION_MAJOR "${PYTHON_VERSION_MAJOR}" CACHE INTERNAL "" FORCE)
```

Comment:
This is just done to cache it, right?

Reply:
yup! I need to be sure we have everything in place and in the same way as it was done in ROOT.